src/App.css

```diff
@@ -1,7 +1,8 @@
 .App {
-	background-color: #000;
+	background-color: rgb(0,0,0);
```

Comment:
I'm not sure why you changed to rgb. Hex code would be better still. Only use rgba (note the a, it stands for alpha channel/opacity) if needed. There's no other reason to use rgb.